cmd/notation/common.go

```diff
@@ -31,27 +31,36 @@ var (
 		fs.StringVarP(p, flagPassword.Name, flagPassword.Shorthand, "", flagPassword.Usage)
 	}
 
-	flagPlainHTTP = &pflag.Flag{
-		Name:     "plain-http",
-		Usage:    "registry access via plain HTTP",
+	flagInsecureRegistry = &pflag.Flag{
+		Name:     "insecure-registry",
+		Usage:    "use HTTP protocol while connecting to registries. Use it only for testing purposes. (alias: --plain-http)",
```

Comment:
```suggestion
		Usage:    "use HTTP protocol while connecting to registries. Should be used only for testing",
```

Reply:
Per our recent discussion around the Notation security best practice, we could remove the alias `--plain-http` entirely in RC.6.